src/Models/TerminusModel.php

```diff
@@ -5,13 +5,16 @@
 use Pantheon\Terminus\Collections\TerminusCollection;
 use Pantheon\Terminus\Request\RequestAwareInterface;
 use Pantheon\Terminus\Request\RequestAwareTrait;
+use Robo\Common\ConfigAwareTrait;
+use Robo\Contract\ConfigAwareInterface;
 
 /**
  * Class TerminusModel
  * @package Pantheon\Terminus\Models
  */
-abstract class TerminusModel implements RequestAwareInterface
+abstract class TerminusModel implements ConfigAwareInterface, RequestAwareInterface
 {
+    use ConfigAwareTrait;
```

Comment:
Rather than using Robo's ConfigAwareTrait, you might want to define your own subclass of it, like Drush does:

https://github.com/drush-ops/drush/blob/master/src/Config/ConfigAwareTrait.php

This allows you to declare that you return a TerminusConfig rather than a Robo Config, which helps with IDE completion et. al.

Reply:
The Travis tests are failing because Pantheon\Terminus\Models\User still uses ConfigAwareTrait, and now TerminusModel does as well. This appears to be okay for PHP 7+, but not for PHP 5.